cloudinit/config/cc_mounts.py

```diff
@@ -42,32 +59,35 @@
 with ``filename``, the size of the swap file with ``size`` maximum size of
 the swap file if using an ``size: auto`` with ``maxsize``. By default no
 swap file is created.
+"""
 
-**Internal name:** ``cc_mounts``
-
-**Module frequency:** per instance
-
-**Supported distros:** all
-
-**Config keys**::
-
+example = dedent(
+    """\
     mounts:
         - [ /dev/ephemeral0, /mnt, auto, "defaults,noexec" ]
         - [ sdc, /opt/data ]
         - [ xvdh, /opt/data, "auto", "defaults,nofail", "0", "0" ]
     mount_default_fields: [None, None, "auto", "defaults,nofail", "0", "2"]
     swap:
-        filename: <file>
-        size: <"auto"/size in bytes>
-        maxsize: <size in bytes>
-"""
-
-import logging
-import os
-import re
-from string import whitespace
-
-from cloudinit import subp, type_utils, util
+        filename: /my/swapfile
```

Comment:
In lines 66-72, is it worth removing quotes around options that don't need it (or possibly putting quotes around everything?). I'm just noticing the inconsistency between `auto` and `"auto"`